scripts/apps/authoring/inline-comments/inline-comments.scss

```diff
@@ -1,48 +1,11 @@
 .sd-widget.inline-comments {
     .notification-list {
-        position: absolute;
-        bottom: 110px;
         top: 0;
-        left: 0;
         right: 0;
+        left: 0;
```

Comment:
If you removed `position:absolute` shouldn't `top` `left` and `right` also be removed?

Reply:
You're right, but it seems `.notification-list` is always `position: absolute`. I could overwrite this or make my own `.comment-list` but I think it's better to reuse it for now and then refactor `.notification-list` if necessary.